tests/saw/sike_r1/proof/sidh.saw

```diff
@@ -29,6 +29,14 @@ let fp2_encode_spec = do {
     crucible_points_to encp (tm enc);
 };
 
```

Comment:
Since this spec will be used as an assumption, it helps to add a comment describing what will be assumed. In this spec, you are assuming that this function always returns `0` and has no side effects when called with pointers to two disjoint memory regions.